docs/ruleset_functions/get_baseline_system_types.md

```diff
@@ -0,0 +1,199 @@
+|----------------------------------|-----------------------------------------------------------------|--------------|--------------|-----------------------------------------|---------------------------------------------------|-------------------------------------------|
+| Sys-1 PTAC                       | Packaged Terminal Air Conditioner                               | DX           | Boiler       | Sys-1a: CV FCU, PCHW w/ Boiler          | Sys-1b: PTAC w/ DX and PHW                        | Sys-1c: CV FCU with PHW and PCHW          |
+| Sys-2 PTHP                       | Packaged Terminal Heat Pump                                     | DX           | HP           | Sys-1a: CV FCU, PCHW w/ Boiler          | Sys-1b: PTAC w/ DX and PHW                        | Sys-1c: CV FCU with PHW and PCHW          |
+| Sys-3 PSZ-AC                     | Packaged Rooftop Air Conditioner                                | DX           | Furnace      | Sys-3a: CV SZ AHU, PCHW w/ Furnace      | Sys-3b: PSZ w/ DX and PHW                         | Sys-3c: CV SZ AHU with PHW and PCHW       |
+| Sys-4 PSZ-HP                     | Packaged Rooftop Heat Pump                                      | DX           | HP           | Sys-3a: CV SZ AHU, PCHW w/ Furnace      | Sys-3b: PSZ w/ DX and PHW                         | Sys-3c: CV SZ AHU with PHW and PCHW       |
+| Sys-5 Package VAV with Reheat    | Packaged Rooftop VAV with Reheat                                | DX           | Boiler       | Sys-7a: VAV with Reheat, PCHW w/ Boiler | Sys-5b: Packaged VAV with Reheat, w/ DX and PHW   | Sys-7c: VAV with Reheat with PHW and PCHW |
+| Sys-6 Package VAV with PFP Boxes | Packaged Rooftop VAV with Parallel Fan-Powered Boxes and Reheat | DX           | ER           | Sys-8a: VAV with PFP Boxes, PCHW w/ ER  | Sys-6b: Package VAV with PFP Boxes, w/ DX and PHW | Sys-8c: VAV with PFP Boxes, with PHW and PCHW |
+| Sys-7 VAV with Reheat            | VAV with Reheat                                                 | CHW          | Boiler       | Sys-7a: VAV with Reheat, PCHW w/ Boiler | Sys-7b: VAV with Reheat, w/ CHW and PHW           | Sys-7c: VAV with Reheat with PHW and PCHW |
+| Sys-8 VAV with PFP Boxes         | VAV with Parallel Fan-Powered Boxes and Reheat                  | CHW          | ER           | Sys-8a: VAV with PFP Boxes, PCHW w/ ER  | Sys-8b: VAV with PFP Boxes, w/ CHW and PHW        | Sys-8c: VAV with PFP Boxes, with PHW and PCHW |
+| Sys-9 HV gas-fired               | Heating and Ventilation                                         | None         | Furnace      | Sys-9: HV (No Changes)                  | Sys-9b: HV (PHW)                                  | Sys-9b: HV (PHW)                          |
+| Sys-10 HV electrical             | Heating and Ventilation                                         | None         | ER           | Sys-9: HV (No Changes)                  | Sys-9b: HV (PHW)                                  | Sys-9b: HV (PHW)                          |
```

Comment:
For air-seide system with PCHW oly, I think it should say Sys-10 and not Sys-9

Reply:
Agreed and updated